### DISCUSSION TITLE
技巧分享：Saladict 侧边栏模式增强——Win10 结合窗口管理器 FancyZones 创建 Saladict 专属的窗口布局

### DISCUSSION THREAD

**Discussion by hqyyqh** [crimx/ext-saladict#593] [Show and tell]
## 先看效果演示
![1](https://user-images.githubusercontent.com/38676455/70790788-bbd62c80-1dd0-11ea-903d-e41abb1e9df3.gif)

## Sladict 自带侧边栏模式的不足与需求-

- **不足：** **侧边栏模式不支持外部应用组合**。独立查词窗口仅能与 Chrome 浏览器主窗口进行组合，不支持外部应用的自动分屏
- **需求：** **独立查词窗口与外部应用的固定翻译布局**。Win10 的分屏功能很好用，但是每次都是等分，经常需要手动调整比例。
![4](https://user-images.githubusercontent.com/38676455/70790838-d3151a00-1dd0-11ea-8bbc-cfd23e0061c5.gif)

**在推荐使用独立查词窗口与外部阅读器搭配后，有一个能够与外部应用配合的固定侧边栏模式将会便利很多**

## 借助 FancyZones 实现设置

FancyZones 是微软开源软件 [PowerToys](https://github.com/microsoft/PowerToys/releases) 的一部分，详细介绍和使用可以看[这里](https://mp.weixin.qq.com/s/vsa0xqwG2JBCa87Tr_5cAw)。下面简述一下设置过程

> Step by Step 的设置步骤就不细说啦

1. 自定义 Saladict 侧边查词的窗口布局

![2](https://user-images.githubusercontent.com/38676455/70790878-e32cf980-1dd0-11ea-9964-2c509da802e9.gif)

2. 选中自己设置好的布局，按住 Shift 拖动窗口

![3](https://user-images.githubusercontent.com/38676455/70790892-e88a4400-1dd0-11ea-8909-ae437b3990a7.gif)

> 后话：Saladict 是一款非常纯粹而功能丰富的翻译插件，而且也给了与其他软件很大的配合交互空间。就是因为这样，偶尔给人惊喜哈哈😁

**Comment by crimx** [comment]
太强了 Orz 真是神器达人......

**Comment by KnIfER** [comment]
怎么做到的，pdf阅读器可以配合 Saladict ?

**Comment by crimx** [comment]
安装时沙拉查词有弹出说明吧？ <https://saladict.crimx.com/native.html>

**Comment by crimx** [comment]
@hqyyqh 小铅笔笔这个分享会挂到知乎上吗？我想收到专栏上，百度对 Gayhub 权重太低了，很多朋友会看不到。如果你没空发我可以整理上去。

**Comment by KnIfER** [comment]
怎么像 quicker 那样调用Saladict,
```
chrome.exe --app=chrome-extension://cdonnmffkdaoajfknoeeecmchibpmkmg/quick-search.html?word=123123
```
不管用啊，总是搜索剪贴板内容？

**Comment by crimx** [comment]
上面分享的[页面](https://saladict.crimx.com/native.html)已经列举了原理。目前仅支持剪贴板作为中介。

**Comment by KnIfER** [comment]
会了，通过插件整合到 Acrobat 的右键菜单里去了

url 作为媒介的话，似乎无法只开一个独立窗口吧。

p.s. 发现取消“启用划词”开关后，连带右键菜单也不能用了。还有，希望能够用右键菜单打开独立窗口。

**Comment by hqyyqh** [comment]
> @hqyyqh 小铅笔笔这个分享会挂到知乎上吗？我想收到专栏上，百度对 Gayhub 权重太低了，很多朋友会看不到。如果你没空发我可以整理上去。

太好了，那就麻烦大大整理了😁。大屏用户结合着 FancyZones  使用确实舒服。
